identifier/vite.config.js

```diff
@@ -1,6 +1,31 @@
-import { defineConfig, splitVendorChunkPlugin } from 'vite';
-import react from '@vitejs/plugin-react';
-import checker from 'vite-plugin-checker';
+import { defineConfig, splitVendorChunkPlugin } from "vite";
+import react from "@vitejs/plugin-react";
+import checker from "vite-plugin-checker";
+import legacy from "@vitejs/plugin-legacy";
+
+
+const htmlBuildPlugin = () => {
```

Comment:
This plugin needs a better name - please use a name which actually describes what the plugin is doing. I suggest `addScriptCSPNoncePlaceholderPlugin` or something in that sense.

Reply:
Renamed this with the suggested name and also updated the name inside the plugin declaration.